tests/conversion/converters/bootstrap_test.py

```diff
@@ -102,36 +102,37 @@ def test_scripts_are_executed_in_correct_order(self, *args, **kwargs):
                 'sql/filter/water/010_water_a_insert_table.sql',
                 'sql/filter/water/020_water_b_insert_table.sql',
                 'sql/filter/water/030_water_l_insert_table.sql',
-                'sql/filter/create_view/000_view_address.sql',
-                'sql/filter/create_view/001_view_adminarea.sql',
-                'sql/filter/create_view/002_view_boundary.sql',
-                'sql/filter/create_view/003_view_building.sql',
-                'sql/filter/create_view/004_view_geoname_l.sql',
-                'sql/filter/create_view/005_view_geoname_p.sql',
-                'sql/filter/create_view/006_view_landuse_a.sql',
-                'sql/filter/create_view/007_view_military_a.sql',
-                'sql/filter/create_view/008_view_military_p.sql',
-                'sql/filter/create_view/009_view_misc_l.sql',
-                'sql/filter/create_view/010_view_natural_a.sql',
-                'sql/filter/create_view/011_view_natural_p.sql',
-                'sql/filter/create_view/012_view_nonop_l.sql',
-                'sql/filter/create_view/013_view_pow_a.sql',
-                'sql/filter/create_view/014_view_pow_p.sql',
-                'sql/filter/create_view/015_view_poi_a.sql',
-                'sql/filter/create_view/016_view_poi_p.sql',
-                'sql/filter/create_view/017_view_railway_l.sql',
-                'sql/filter/create_view/018_view_road_l.sql',
-                'sql/filter/create_view/019_view_route_l.sql',
-                'sql/filter/create_view/020_view_traffic_a.sql',
-                'sql/filter/create_view/021_view_traffic_p.sql',
-                'sql/filter/create_view/022_view_transport_a.sql',
-                'sql/filter/create_view/023_view_transport_p.sql',
-                'sql/filter/create_view/024_view_utility_a.sql',
-                'sql/filter/create_view/025_view_utility_p.sql',
-                'sql/filter/create_view/026_view_utility_l.sql',
-                'sql/filter/create_view/027_view_water_a.sql',
-                'sql/filter/create_view/028_view_water_p.sql',
-                'sql/filter/create_view/029_view_water_l.sql',
+                'sql/filter/create_view/0110_view_address.sql',
```

Comment:
As I had to renumber anyway, I though I'd
* group the layers
* allow for insertions within the groups

as there are 19 groups, the groups require two decimal digits. I used another two decimal digits for the numbering within the group, so that I can assign the more significant of them (the "ten"s) sequentially while still allowing for later insertions in-between using the last (and least significant) digit.

The question is, though: Do we need a decimal prefix for the view-creation SQL scripts at all? They should all be independent of each other, so we could just create them in lexicographic order of their un-prefixed script names. (We already almost do, there are just some few swaps within the "groups".)